<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinduWindow</class>
 <widget class="QMainWindow" name="LinduWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinduWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="prog_check">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="lindu_tree">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Lindu</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Programs</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Traveltime Tomography (BETA)</string>
        </property>
        <property name="flags">
         <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>2D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Earthquakes</string>
        </property>
        <property name="flags">
         <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>Locate</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relocate</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Displays</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Map</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seismo View</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>View</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plot</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable</set>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="lindu_stack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuPrograms">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Programs</string>
    </property>
    <addaction name="action_tomography"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlugin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Plugin</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPrograms"/>
   <addaction name="menuProject"/>
   <addaction name="menuPlugin"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_tomography">
   <property name="text">
    <string>Velocity &amp;Tomography (BETA)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
